### DISCUSSION TITLE
Develop a `radashi-dom` package

### DISCUSSION THREAD

**Discussion by aleclarson** [radashi-org/radashi#197] [Ideas]
A utility library for convenient use of DOM APIs.

### Possible functions
- [measureText](https://gist.github.com/aleclarson/55a580acfa5ba7817d62fb8c9e307228)
- [toPixels](https://gist.github.com/aleclarson/a92f76f7bcd3391f7d19880246164abc)

Feel free to suggest other functions you'd like to see in this package.

**Comment by MarlonPassos-git** [comment]
Suggestion: 
- [waiForElement](https://gist.github.com/MarlonPassos-git/212046ecfd5bc563f5a530c3df7a0212)
